src/mscorlib/src/System/IO/FileSystemInfo.cs

```diff
@@ -185,11 +185,13 @@ public DateTime CreationTime
             }
         
             set {
+#if !FEATURE_CORECLR
```

Comment:
I figured a no-op would probably be fine, but I'm alright with commenting out the whole thing.